### DISCUSSION TITLE
The maintainer can’t push to Pull Requests that they can edit because they don’t have permissions to touch the fork repo

### DISCUSSION THREAD

**Discussion by ParadoxV5** [community/community#84154] [Pull Requests]
### Select Topic Area

Bug

### Body

1. You have repo `Main` with branch `main`
2. Contributor forks `Main` as `Fork` and creates branch `feature`
3. Contributor opens PR `#1` to merge `Fork/feature` to `Main/main`
4. Contrubitor “allow[s] edits and access to secrets by maintainers”
5. You opens `Main/pr/1` in GitHub Desktop and make a commit
6. You __can’t__ push your commit to `Main/pr/1` with GitHub Desktop or git: Error 403

**Comment by bonizario** [comment]


**Reply by ParadoxV5** [reply]
Oy, this gives the very same 403.

**Comment by vivodi** [comment]
"I followed every step of [this official tutorial](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork?platform=windows) exactly, but in the end, I got a 403 error. Currently, I can only edit through the GitHub online editor.
